### PR TITLE
(update) SearchBar.jsx to dynamically display ratings by user. (Update)

### PR DESCRIPTION
**Pull request by ABailey92**
Index.js to include routes

### CODE REVIEW

**Review by BenYoss — APPROVED** [review]

File: client/src/components/SearchBar/searchEntry.jsx [hunk 1]
```diff
@@ -71,11 +73,31 @@ const SearchFeedEntry = ({ show, onClick }) => {
...
+    });
+  };
   const handleChange = (event) => {
     setText(event.target.value);
   };
 
+  const getShows = async () => {
+    const { data } = await axios.get('/shows');
+    setComments(data);
+    console.log(data);
```
Comment (jstrzesz) [inline client/src/components/SearchBar/searchEntry.jsx:94]: remove

File: client/src/components/SearchBar/searchEntry.jsx [hunk 2]
```diff
@@ -131,15 +153,14 @@ const SearchFeedEntry = ({ show, onClick }) => {
...
         ) : <Typography component="legend">Average Rating N/A</Typography> }
         <>
           <Typography component="legend">Your Rating</Typography>
           <div className={classes.root}>
             <Rating
               name={show.name}
-              defaultValue={0}
-              value={value}
+              value={getRating()}
+              readOnly={!!value}
```
Comment (jstrzesz) [inline client/src/components/SearchBar/searchEntry.jsx:163]: why are you using a double bang here?

File: client/src/components/SearchBar/searchEntry.jsx [hunk 3]
```diff
@@ -170,7 +191,17 @@ const SearchFeedEntry = ({ show, onClick }) => {
 
               />
               <br />
-              <Button onClick={async () => { setShowPopUp({}); const { data: { id } } = await axios.get('/user'); console.warn('THIS IS ID', id); await axios.put('/show', { idUser: id, idShow: show.id, comment: text, rating: value }); }} variant="contained" color="primary" href="#contained-buttons">
+
+              <Button
+                onClick={async () => {
```
Comment (jstrzesz) [inline client/src/components/SearchBar/searchEntry.jsx:196]: extract all of this to a separate method that is then called in the onclick

File: client/src/components/app.jsx [hunk 4]
```diff
@@ -160,7 +159,9 @@ const App = () => {
           </a>
         )}
       {getUser()}
-      {/* {getPosts()} */}
+
+      {/* { getPosts()} */}
```
Comment (jstrzesz) [inline client/src/components/app.jsx:163]: remove

File: client/src/components/SearchBar/searchEntry.jsx (hunk 2 above)
Comment (ABailey92) [reply client/src/components/SearchBar/searchEntry.jsx:163]: Eslint does not allow for the use of "unnecessary boolean values". Double bang is used here to check if a value is present if so the component will be read only and non editable.

**Review by ZiaTahir504 — APPROVED** [review]

**Pull request merged**